sbtc/src/events.rs

```diff
@@ -640,7 +655,8 @@ impl RawTupleData {
             // amount of sats by us.
             fee: u64::try_from(fee).map_err(EventError::ClarityIntConversion)?,
 
-            sweep_block_hash: BitcoinBlockHash::from_le_bytes(sweep_block_hash),
+            sweep_block_hash: BurnchainHeaderHash::from_bytes(&sweep_block_hash)
+                .ok_or_else(|| EventError::BurnchainHashConversion(sweep_block_hash.into()))?,
```

Comment:
We should keep what was here, it is correct. No need to use `BurnchainHeaderHash` anymore.